canonical-web-and-design/snapcraft.io
# Add stores side nav to admin

## Done

- Added side nav to admin view with list of available stores, according to [the design](https://app.zeplin.io/project/60c863a20f844214a2a97f3c/screen/61112e2dcd78051225df109c)

## How to QA
- Visit the demo in your browser, link commented below
  - Or you can run the project locally using the [dotrun](https://snapcraft.io/dotrun) snap with `$ dotrun` and view it in your web browser at: https://snapcraft-io-3668.demos.haus/admin/ahnuP3quahti9vis8aiw/snaps
  - If your demo doesn't work, read more about the [demoservice](https://discourse.ubuntu.com/t/demoservice/16862)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- On different viewpoints, interact with the sidenav, make sure it matches the design and works as expected

## Issue / Card
Fixes #3649
